src/screens/ConnectedDappsSheet.js

```diff
@@ -48,8 +91,15 @@ export default function ConnectedDappsSheet() {
         {lang.t('walletconnect.connected_apps')}
       </SheetTitleWithPadding>
       <Divider color={colors.rowDividerExtraLight} inset={[0, 19]} />
-      <ScrollableItems length={mostRecentWalletConnectors.length}>
+      <ScrollableItems length={numOfRows}>
         <Row height={4} />
+        {sessions.map(session => (
```

Comment:
would be nice to still keep all sessions sorted chronologically regardless of V1 vs V2

Reply:
It's a good idea, but v1 connectors don't have a timestamp, they're sorted by the "handshake ID". Think there's another way to do this?